EXT_PRAYTRACE.md

```diff
@@ -0,0 +1,188 @@
+Domain-Specific Naming Conventions for Photon Raytracing Codes
+===========================================================================
+
+openPMD extension name: `PRAYTRACE`
+
+
+Introduction
+------------
+
+This extension to the openPMD addresses the domain-specific requirements
+met in raytracing codes (e.g. OASYS).
+
+Particle Based Records (Fields)
+---------------------------
+
+### Root attributes
+
+- `openPMD`: `0.0.1`
+- `openPMDextension`: `PRAYTRACE`
+- `author`: `Aljosa Hafner <aljosa.hafner@ceric-eric.eu>`
+- `date`: `10.10.2019 00:00:00 +0100`
+
+### Attributes for the Group 'particlesPath'
+
+	- `nRays`
+		- type: *(uintX)*
+		- description: Number of initialised rays
+		- scope: *required*
+	
+	- `rayPositions`
```

Comment:
general:
my suggestion was to make one datagroup for each ray, see [here](https://github.com/PaNOSC-ViNYL/ViNYL-project/wiki/Meeting-notes#deliverable-51-1).  that would be more in line with the original standard and also ensure that many of the tools around openpmd (viewer, parsers, etc) will continue to work.

please add information on the units (unitSI) and unitDimension to all records. even if the unit is just 1